content/techniques/file-upload.md

```diff
@@ -4,6 +4,14 @@ To handle file uploading, Nest provides a built-in module based on the [multer](
 
 > warning **Warning** Multer cannot process data which is not in the supported multipart format (`multipart/form-data`). Also, note that this package is not compatible with the `FastifyAdapter`.
 
+#### Use with Express (default)
+
+To work properly with Typescript, its types must be installed:
```

Comment:
```suggestion
First install the types of the required package to properly work with Typescript:
```

Reply:
I think either of these is fine here. Maybe we could use something like

```suggestion
If you want to use the `Express.Multer.File` type for `@UploadedFile()` files, it is necessary to install the types:
```

to show that this is a completely optional step and only necessary if the dev wants to use this type.

Reply:
@jmcdo29  i think if we really have to make a change, I'd say use yours as a change, ok?